dex-it/src/test/resources/logback-test.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <configuration>
-    <property name="pattern" value="%date %-5level [%.25thread] %logger{26} - %msg%n%ex"/>
+    <property name="pattern" value="%date{ISO8601,UTC} %-5level [%.25thread] %logger{26} - %msg%n%ex"/>
```

Comment:
Can we somehow log time inside conteiners with UTC+3?

Reply:
The time in Docker and in test logs had different timezone, and I chose UTC, because we have UTC in all tools we use.